src/interfaces/multipliers/INFTMultiplier.sol

```diff
@@ -16,4 +16,8 @@ interface INFTMultiplier is IMultiplier {
     /// @param _user The address of the _user to check
     /// @return True if the _user owns at least one NFT, false otherwise
     function hasNFT(address _user) external view returns (bool);
+
+    /// @notice Updates the multiplying factor for a specific user
+    /// @param _user The address of the user to update the multiplying factor for
+    function updateMultiplyingFactor(address _user) external;
```

Comment:
This same abstract function is already declared in the `IMultiplier` interface that this interface inherits. I think this is redundant and be removed.